### ISSUE TITLE
Setup - Updates & Missing Libraries

### ISSUE THREAD

**Issue by kiritigowda** [GPUOpen-ProfessionalCompute-Libraries/MIVisionX#969]
OpenMP - 

* Ubuntu -- libomp-dev

**Comment by kiritigowda** [comment]
Add `--debug` option 

```
sudo apt update
sudo apt install build-essential texinfo
sudo apt-get install libgmp-dev
cd ~
wget https://ftp.gnu.org/gnu/gdb/gdb-12.1.tar.gz
tar -xvzf gdb-12.1.tar.gz
cd gdb-12.1
./configure --with-python
make CXXFLAGS="-static-libstdc++"  -j16
sudo make install -j16
```

**Comment by kiritigowda** [comment]
Setup -- U22.04 discontinued packages 

```
python-dev python-numpy libdc1394-22-dev
```

Add
```
lib boost-all-dev
```

**Comment by kiritigowda** [comment]
FFMPEG issue is related -- #985 
```
apt install libass-dev
```

**Comment by kiritigowda** [comment]
Fixed with #1013
